scripts/imagebuild.sh

```diff
@@ -54,6 +54,8 @@ echo "Pushing image with tag '${TAG}'."
 docker push ${REGISTRY}${REPO}/federation-v2:${TAG}
 
 if ! [ -z "${LATEST}" ]; then
+
+   docker tag ${REGISTRY}${REPO}/federation-v2:${LATEST}
```

Comment:
This doesn't look correct - a source is needed as well:

```
docker tag ${REGISTRY}${REPO}/federation-v2:${LATEST} ${REGISTRY}${REPO}/federation-v2:${TAG}
```

Reference: https://docs.docker.com/engine/reference/commandline/tag/

Reply:
Yep. You'll need to put the source image first:

```
docker tag ${REGISTRY}${REPO}/federation-v2:${TAG} ${REGISTRY}${REPO}/federation-v2:${LATEST}
```